docker/kubernetes/kustomize/env-dev

```diff
@@ -23,6 +23,7 @@ REDIS_PORT=6379
 WS_PORT=3002
 
 # Root URL
+REACT_APP_API_URL=http://localhost:3000
```

Comment:
I tested & deployed it on my staging environment. 
Setting `API_ROOT_URL` won't update `REACT_APP_API_URL` on the web client. When checking the injected `window._env_` on the dev tools, it was `http://localhost:3000` instead of my `API_ROOT_URL`.
There's probably a bug on the ENV injection code but I haven't become familiar with the project yet.

Setting `REACT_APP_API_URL` on the env-dev fixed the issue.

Reply:
Did you execute this script to set the Web environment variables in the window object?
https://github.com/novuhq/novu/blob/00e007606c29de651e2b255a93c5127b32ce785d/apps/web/package.json#L11

The Docker image is using API_ROOT_URL and it then executes the environment setup to load properly in the window object the value of REACT_APP_API_URL.
https://github.com/novuhq/novu/blob/00e007606c29de651e2b255a93c5127b32ce785d/docker/local/deployment/docker-compose.yml#L98

Not saying that this change might be needed, but the intentions in the setup is to use `API_ROOT_URL` all over.

@Cliftonz am I misleading here and @SergeiMeza is right about this env needed for the k8s implementation?

Reply:
Yes, because the kustomize config generator works differently. If you look here:
https://github.com/novuhq/novu/blob/next/docker/kubernetes/kustomize/kustomization.yaml#L31
the config takes the env-dev file and directly places it in to a configmap which then get pulled from.
Thus there is not the REACT_APP_API_URL as @SergeiMeza has mentioned.
If we add it into the env-dev file it should work.